ThomasNels/EmotiGame
# Add navbar styling, template pages for games, setup admin page, added requirements.txt, updated README

# Description
I updated the navbar to look better, but I still need to add things such as images and better styling based on the desired appearance. Made templates for each game in the user study and added them to the navbar. Set up the main page and styling to be ready for database functionality. Created a requirements.txt file for installing needed packages and updated the README.md file accordingly. 

# Requirements
The website should be run from a new virtual environment using the requirements.txt file to ensure all dependencies are included. The website should be tested to ensure the navbar is displayed correctly and goes to the right page. The admin page should be displayed correctly and allow users to select files in the table, but not be able to use the buttons as intended yet. 


# Issues Referenced

- Fixes #4 

# Documentation update
Link to a change in documentation (if any)

# Notes for Reviewer
File functions will be added for button functionality on the admin page as well as authenticating the user once the database is implemented. This will also include displaying files in the database rather than temporary placeholders. 

# Testing
- Follow the steps in the README.md located in the website folder to ensure it is accurate and can properly display the website.
- Create two users in the local database using the following: 
       If using Linux type into the terminal
`sudo -u psql <database username>` 
       Then run the following commands to insert one admin user and one normal user (test) into the database
`INSERT INTO user_information VALUES
(1, 'admin', 'password', 'admin@gmail.com', 'Admin'),
(2, 'test', 'password', 'test@gmail.com', 'Test User');`
- Login as the admin user
- Click on each navbar item. The current tab should have an outline and if the mouse hovers over a tab another outline should be displayed. Ensure that each webpage displays correctly.
- Click the link label "Admin"
- Click on one of the "Select" boxes within the table. The selected box should display a checkmark. 
- Logout and go to http://127.0.0.1:5000/admin 

## Results

- A new user should be able to follow the README.md and be able to launch the website. 
- Each page should display correctly when using the navbar. 
- When logged out the admin tab on navbar should be hidden
- When logged out and trying to access the admin page by the URL you should be redirected to the landing page and a message should appear at the top of the screen. 

## Feature Tests
TODO

# Checklist to allow merge
- Software

## Screenshots:
